pkgs/top-level/all-packages.nix

```
@@ -3674,6 +3674,8 @@ with pkgs;
 
   dpt-rp1-py = callPackage ../tools/misc/dpt-rp1-py { };
 
+  doomsday-engine3 = callPackage ../by-name/do/doomsday-engine/3.x.nix { };
```

```suggestion
  doomsday-engine2 = callPackage ../by-name/do/doomsday-engine/2.x.nix { };
```
wouldn't the newer version be preferrable as the default over an older one?

---

Doomsday 3 is still very much WIP as far as I can tell, such that it's even a minor miracle that it even works. There's no official builds available and Nixpkgs will possibly be the only place binaries for Doomsday 3 exist outside of the author's dev environment. Therefore I don't think it's sound to offer it as the default option even though it appears to work just fine for me

---

makes total sense